openwisp_notifications/types.py

```diff
@@ -17,9 +17,26 @@
         'email_notification': True,
         'web_notification': True,
     },
+    'general_message': {
```

Comment:
@pandafy @Dhanus3133 what do you think about changing this to just `generic`?

Rather than general, it's generic. This is a generic notification type that we are adding to allow any module to send information to users for any need that may arise.

The fact that it's a message it's also kind of implied, so for these reasons I would just flag it `generic` for the sake of brevity.